packages/@lwc/integration-karma/scripts/karma-plugins/lwc.js

```diff
@@ -108,7 +108,12 @@ function createPreprocessor(config, emitter, logger) {
             const location = path.relative(basePath, file.path);
             log.error('Error processing “%s”\n\n%s\n', location, error.stack || error.message);
 
-            done(error, null);
+            if (process.env.KARMA_MODE === 'watch') {
+                log.error('Ignoring error in watch mode');
+                done(null, content); // just pass the untransformed content in for now
```

Comment:
This ends up giving an error in the browser because you can't have `import` statements inside a non-module. That's fine – it's better than crashing.

I did verify that if you fix the syntax error, you get the updated file contents.